pkg/registry/extensions/daemonset/strategy.go

```diff
@@ -70,6 +70,16 @@ func (daemonSetStrategy) PrepareForUpdate(ctx genericapirequest.Context, obj, ol
 	// update is not allowed to set TemplateGeneration
 	newDaemonSet.Spec.TemplateGeneration = oldDaemonSet.Spec.TemplateGeneration
 
+	// Update is not allowed to set Spec.Selector for all groups/versions except extensions/v1beta1.
```

Comment:
sure! i removed the comment as it was not applicable anymore, but forgot to remove the comment link.